ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```
@@ -219,8 +228,14 @@ cat /tmp/all.yml
 envsubst < /tmp/all.yml > /tmp/all-updated.yml
 
 # Copy the ssh key to the bastion host
-OCPINV=$QUADS_INSTANCE/instack/$LAB_CLOUD\_ocpinventory.json
-bastion2=$(curl -sSk $OCPINV | jq -r ".nodes[0].name")
+if [[ "${OCP_INVENTORY_OVERRIDE}" == "true" ]]; then
+   scp -q ${SSH_ARGS} root@${bastion}:${OCP_INVENTORY_PATH} /tmp/inventory_${LAB_CLOUD}.json
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)